src/coordination/include/nuraft/coordinator_cluster_state.hpp

```diff
@@ -77,7 +77,7 @@ void from_json(nlohmann::json const &j, CoordinatorInstanceState &instance_state
 // Source of truth since it is modified only as the result of RAFT's commiting
 class CoordinatorClusterState {
  public:
-  explicit CoordinatorClusterState(CoordinatorInstanceInitConfig const &config);
+  CoordinatorClusterState() = default;
```

Comment:
The config is now saved in the instance, right?

Reply:
yes